packages/cli/snapshots/integration/generators/relation.belongs-to.integration.snapshots.js

```diff
@@ -46,12 +50,12 @@ export class OrderClassRepository extends DefaultCrudRepository<
   typeof OrderClass.prototype.orderNumber
 > {
 
-  public readonly customerClass: BelongsToAccessor<CustomerClass, typeof OrderClass.prototype.orderNumber>;
+  public readonly customerClassCustNumber: BelongsToAccessor<CustomerClass, typeof OrderClass.prototype.orderNumber>;
```

Comment:
hmm what does `Cust` mean 🤔 ?

Reply:
`CustNumber` is the name of the id property of `customerClass` model.

Reply:
I change the generator to generate `customerClassId` instead to match with our code in `/repository/relation`